applications/templates/convert_mentor.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'base_tabs.html' %}
+
+{% load bootstrap3 %}
+{% block head_title %}Application{% endblock %}
+
+{% block panel %}
+
+  <div>
+  <h1 style="text-align: center">Please read this!</h1>
+  <p style="text-align: center">You can apply to come at {{ h_name }} as mentor. Be aware that if you choose to be a mentor,
+                    your hacker application will be <b>removed</b>. This action can't be reverted</p>
+  <form enctype="multipart/form-data" method="post" action="">
+    {% csrf_token %}
+    <button class="btn btn-success btn-block">I want to become mentor</button>
+    <a class="btn btn-danger btn-block" href={% url 'dashboard' %}>Go back</a>
+  </form>
+  </div>
+
+{% endblock %}
```

Comment:
Why do we allow this?

Reply:
It is usual that hackers apply and then they want to change to mentors when they are doubtfull.

Reply:
Mmmmm this seems easy to exploit by hackers that have not been invited to hackathon

Reply:
No, it's not, hackers status can go like this pending -> doubtfull -> Invalid. Invalid applications are the only ones that can convert to mentors. And then they have to be reviewed by the team as well.

Reply:
I see. It may be worth adding a new diagram for all status/user types to the README (can be done in a followup PR, pls create an issue for this)